<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamPage</class>
 <widget class="QWidget" name="ExamPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QFrame" name="leftPanel">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="titlePlaceholder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="thumbnailImageListPlaceholder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>4</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="rightPanel">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(208, 208, 208);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContent">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>382</width>
            <height>770</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="studyPlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Study Control</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="exposurePlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(208, 208, 208);</string>
             </property>
             <property name="title">
              <string>Exposure Control</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="applicationControlGroupBox">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Application</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="bodySizePlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(208, 208, 208);</string>
             </property>
             <property name="title">
              <string>Body Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="endExposurePlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(208, 208, 208);</string>
             </property>
             <property name="title">
              <string>End Exposure</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="focalSpotPlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(208, 208, 208);</string>
             </property>
             <property name="title">
              <string>GroupBox</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="aecControlPlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(208, 208, 208);</string>
             </property>
             <property name="title">
              <string>AEC Control</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="generatorControlPlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Generator</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="detectorControlWidget">
             <property name="styleSheet">
              <string notr="true">QTabWidget::pane {
    border: 1px solid rgb(220, 220, 220); /* Custom border color */
    background-color: rgb(83, 83, 83);    /* Background inside the box */
}

QTabBar::tab {
    background-color: rgb(83, 83, 83);  /* Tab color */
    color: white;                       /* Tab text color */
    padding: 5px;
    border: 1px solid rgb(60, 60, 60);
}

QTabBar::tab:selected {
    background-color: rgb(60, 60, 60);  /* Slightly darker on selection */
}

QTabBar::tab:hover {
    background-color: rgb(100, 100, 100);  /* Optional hover effect */
}
</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="firstDetectorPlaceholder">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <attribute name="title">
               <string>1st Detector</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="secondDetectorPlaceholder">
              <attribute name="title">
               <string>2nd Detector</string>
              </attribute>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="buckyControlPlaceholder">
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Bucky Position</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
